target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,16 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus,shield-router
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
+  CMDLINE := $(ITUSROUTER_CMDLINE)
+  IMAGE/sysupgrade.tar/squashfs :+ append-metadata | combined-image
```

Comment:
Does ":+" exist?

Reply:
Nope.. and Now I have to test to see if I even need the line ;p